tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -66,3 +66,46 @@ Feature: display image in media viewer on the webUI
     And user "user1" has logged in using the webUI
     When the user views the file "testavatar.jpg" in the media viewer by clicking on the file name using the webUI
     Then the file "testavatar.jpg" should be displayed in the media viewer webUI
+
+  Scenario: preview of image in file list view
+    Given user "user1" has uploaded file "testavatar.jpg" to "testavatar.jpg"
+    And user "user1" has logged in using the webUI
+    When the user browses to the files page
+    Then the preview image of file "testavatar.jpg" should be displayed in the file list view on the webUI
+
+  Scenario: preview of file in file list view when previews is disabled
+    Given the property "disablePreviews" of "options" has been set to true in web config file
+    And user "user1" has uploaded file "testavatar.jpg" to "testavatar.jpg"
+    And user "user1" has logged in using the webUI
+    When the user browses to the files page
+    Then the preview image of file "testavatar.jpg" should not be displayed in the file list view on the webUI
+
+  Scenario: file list view image preview in public share
+    Given user "user1" has uploaded file "testavatar.jpg" to "simple-empty-folder/testavatar.jpg"
+    And user "user1" has created a public link with following settings
+      | path | simple-empty-folder |
+    When the public uses the webUI to access the last public link created by user "user1"
+    Then the preview image of file "testavatar.jpg" should be displayed in the file list view on the webUI
+
+  Scenario: file list view image preview in public share when previews is disabled
+    Given the property "disablePreviews" of "options" has been set to true in web config file
+    And user "user1" has uploaded file "testavatar.jpg" to "simple-empty-folder/testavatar.jpg"
+    And user "user1" has created a public link with following settings
+      | path | simple-empty-folder |
+    When the public uses the webUI to access the last public link created by user "user1"
+    Then the preview image of file "testavatar.jpg" should not be displayed in the file list view on the webUI
+
+  Scenario: file list view image preview in file share
```

Comment:
disable auto accept share and set a share receive folder, that way we can use this scenario in ocis too